src/site/content/en/blog/better-responsiveness-metric/index.md

```diff
@@ -0,0 +1,508 @@
+that the `pointerdown`, `pointerup`, and `touchend` are dispatched quickly, whereas the `mousedown`,
+`mouseup`, and `click` wait for the delay before being dispatched. This means that if we only looked
+at `pointerdown` and `pointerup` then we’d miss the duration from the synthetic events, which is
+large due to the browser tap delay and should be included. So we should measure `pointerdown`,
+`pointerup`, and `click` to cover the full interaction. {% endDetails %}
+
+##### Drag
+
+We decided to include dragging as well since it has similar associated events and since it generally
+causes important UI updates to sites. But for our metric we intend to only consider the drag start
+and the drop - the initial and final parts of the drag. This is to make it easier to reason about as
+well as make the latencies comparable with the other interactions considered. This is consistent
+with our decision to exclude continuous events such as `mouseover`.
+
+We’re also not considering drags implemented via the [Drag and Drop
+API](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API) because they only work
+on desktop.
+
+##### Scrolling
+
+One of the most common forms of interacting with a site is via scrolling. For our new metric, we’d
+like to measure the latency for the initial scrolling interaction of the user. In particular, we
+care about the initial reaction of the browser to the fact that the user requested a scroll. In
+particular, we will not cover the whole scrolling experience. That is, scrolling produces many
+frames, and we’ll focus our attention on the initial frame produced as a reaction to the scroll.
+
+Why just the first one? For one, subsequent frames may be captured by a separate smoothness
+[proposal](https://docs.google.com/presentation/d/1VwGIzypntWNosCTXWMsUI6ifw4sEKSRQgnwx3P_wqVg/edit#slide=id.p).
+That is, once the user has been shown the first result of scrolling, the rest should be measured in
+terms of how smooth the scrolling experience feels. Therefore, we think that the smoothness effort
+could better capture this. So, as with FID, we choose to stick to discrete user experiences: user
+experiences that have clear points in time associated with them and for which we can easily compute
+their latency. Scrolling as a whole is a continuous experience, so we do not intend to measure all
+of it in this metric.
+
+So why measure scrolls? The scrolling performance we’ve gathered in Chrome shows that scrolling is
+generally very fast. That said, we still want to include initial scroll latencies in our new metric
+for various reasons. First, scrolling is fast only because it has been optimized so much, because it
+is so important. But there are still ways for a website to bypass some of the performance gains that
+the browser offers. The most common one in Chrome is to force scrolling to happen on the main
+thread. So our metric should be able to say when this happens and causes poor scrolling performance
```

Comment:
Nit: `So out metric should be able to say` --> `We would like the responsiveness metric to clearly detect`